src/Gui/StartupProcess.cpp

```diff
@@ -231,6 +231,7 @@ void StartupPostProcess::execute()
     showMainWindow();
     activateWorkbench();
     checkParameters();
+    hideSplashScreen();
```

Comment:
You might as well call this from `delayedStartup()` slot, at that time event loop just started running.
But all this is just a toying around messy pot of spaghetti which splasher, about dialog, startup process, main window and (gui) application is.